src/test_fixture/metrics.rs

```diff
@@ -57,7 +57,11 @@ impl MetricsHandle {
         // logging is required to import span fields as metric values
         logging::setup();
 
-        let id = thread_rng().gen::<u128>();
+        let id = thread_rng()
+            .sample_iter(&Alphanumeric)
+            .take(8)
```

Comment:
Is 40 bits enough to avoid collisions?  I guess it is.

Reply:
I was thinking about this id as a string where each character can have at most 62 different values.  We could have $62^8$ possible combinations and to get 50% chance of collision we need to run ~14M unit tests in parallel.

Reply:
OK, that's probably OK - collisions aren't necessarily fatal either, they just mix things together.

Reply:
yea, collisions are "fatal" if metrics test collide with any other test - that will make the metrics test fail. I'll keep an eye on them, any id string that has less than 31 characters in it will be a clear win compared to u128. I thought that using 64 bit identifiers should be fine too, that gets us to 4B unit tests running in parallel. If we get to that number, I will happily retire (and change it back to u128)